app/views/root/_error_page.html.erb

```diff
@@ -26,6 +26,12 @@
       </div>
     </article>
   </div>
+
+<% if status_code %>
+  <script type="text/javascript">
+    GOVUK.httpStatusCode = '<%= status_code %>';
```

Comment:
I just used `GOVUK` out of a reflexive dislike of adding anything to `window` - but perhaps `window` is better here if we can't assume that `GOVUK` is available. Will defer to @edds on this.